lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3637,10 +3636,14 @@ namespace {
           if (unannotatedAPIWarningNeeded &&
               Impl.SwiftContext.LangOpts.hasFeature(
                   Feature::WarnUnannotatedReturnOfCxxFrt)) {
-            Impl.addImportDiagnostic(
-                decl,
-                Diagnostic(diag::no_returns_retained_returns_unretained, decl),
```

Comment:
Is `no_returns_retained_returns_unretained` still used elsewhere? If not, maybe it should be removed from `DiagnosticsClangImporter.def`.